node/runtime/pangolin/src/pallets/evm.rs

```diff
@@ -146,22 +146,23 @@ where
 			a if a == addr(7) => Some(Bn128Mul::execute(input, target_gas, context, is_static)),
 			a if a == addr(8) => Some(Bn128Pairing::execute(input, target_gas, context, is_static)),
 			a if a == addr(9) => Some(Blake2F::execute(input, target_gas, context, is_static)),
-			// Darwinia precompiles: 1024+ for stable precompiles, 2048+ for experimental
-			// precompiles
+			// Darwinia precompiles: 1024+ for stable precompiles.
+			// TODO: Change the transfer precompile address after https://github.com/darwinia-network/darwinia-common/issues/1259
 			a if a == addr(21) =>
 				Some(<Transfer<R>>::execute(input, target_gas, context, is_static)),
+			// TODO: Delete EthereumBridge and Sub2SubBridge precompiles in the futures.
 			a if a == addr(23) =>
 				Some(<EthereumBridge<R>>::execute(input, target_gas, context, is_static)),
 			a if a == addr(24) => Some(<Sub2SubBridge<
 				R,
 				ToPangoroMessageSender,
 				bm_pangoro::ToPangoroOutboundPayLoad,
 			>>::execute(input, target_gas, context, is_static)),
-			a if a == addr(25) =>
-				Some(<Dispatch<R>>::execute(input, target_gas, context, is_static)),
-			a if a == addr(27) => Some(<StateStorage<R, StorageFilter>>::execute(
```

Comment:
We can remove this in testnets, but for Crab, let's keep this id for a while as an legacy compatibility concern.

Util all apps remove use of deprecated ids, we can safely remove them then.

Reply:
Let's keep this id for dispatch precompile in pangolin too until Hexlix make their changes, and replacing with new precompiles id.

Reply:
Okay.

Reply:
remove this call on testnets can also cause failure of Helix ci.